.github/workflows/build-desktop-external-reusable.yml

```diff
@@ -19,13 +19,18 @@ jobs:
   build-desktop-app:
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
-    name: "Build Ledger Live Desktop"
-    runs-on: ${{ matrix.os }}
```

Comment:
this is a result of the workflows in `build-desktop-external-reusable.yml` and `build-desktop-reusable.yml` drifting out of sync, as they are partial copy-pasted duplicates of each other. Would love to remove as much duplication as possible (ideally delete the unused workflow)

Reply:
I'm not sure. We don't want to use our internal runners for external contributions. So I think that ubuntu-latest is used for Github runner. Can you double check, please?

Reply:
Good point - fixed